lib/ecto_stream_factory/exceptions.ex

```diff
@@ -30,10 +30,25 @@ defmodule EctoStreamFactory.RepoNotSpecifiedError do
     """
     :repo option is missing. Please configure the factory:
 
-        defmodule #{module} do
+        defmodule #{inspect(module)} do
           use EctoStreamFactory, repo: #{repo_module}.Repo
           ...
         end
     """
   end
 end
+
+defmodule EctoStreamFactory.MissingKeyError do
+  @moduledoc false
```

Comment:
Why suppress exception docs? Even if you feel it does not requires any text in the `@moduledoc` attribute, then just drop the attribute. Setting it to `false` will hide the exception from the docs, which seems unnecessary...